assignments/assignment04.rb

```diff
@@ -12,7 +12,13 @@
 # F[n] -> F[n-2] + F[n-1]
 
 def fib(n)
-  # your implementation here
+  def fib_helper(n, acc1, acc2)
```

Comment:
@planetcohen Michael, I do understand the solution to this problem. This is a tail recursive solution to the fib problem. It was posed as an exercise in a book I was reading (Functional Programming in Scala, copyright 2015 by Manning Publications, Exercise 2.1, page 21). I originally coded the fib function using the classic recursive algorithm. Then I updated to using tail recursion.